antora-playbook.yml

```diff
@@ -61,7 +61,7 @@ asciidoc:
     image-puller-name: Kubernetes Image Puller
     image-puller-name-short: Image Puller
     image-puller-repository-name: kubernetes-image-puller
-    image-puller-serviceaccount-name: k8s-image-puller
+    # image-puller-serviceaccount-name: kubernetes-image-puller
```

Comment:
why is it commented out?

Reply:
just for the record the name of sa is correct - https://github.com/che-incubator/kubernetes-image-puller/blob/master/deploy/openshift/serviceaccount.yaml#L6

Reply:
will delete the `image-puller-serviceaccount-name: kubernetes-image-puller` attribute and put it down in a hardcoded way since this property will hardly change in the future.

Reply:
+1 let's avoid commented attributes that are not needed - it creates a lot of confusion and makes docs hard to maintain

Reply:
Removed. Thx! @ibuziuk